cora/signal/lss.py

```diff
@@ -703,6 +785,40 @@ def process(self, initial_field: InitialLSS, biased_field: BiasedLSS) -> BiasedL
         return final_field
 
 
+# List the models that are available for validation by the classes
+_omega_HI_models = ["Crighton2015", "SKA", "uniform"]
+
+
+def omega_HI(
+    z: FloatArrayLike, model: Literal["Crighton2015", "SKA", "uniform"] = "Crighton2015"
+):
+    """Calculate the HI fraction at specified redshifts.
+
+    Parameters
+    ----------
+    z
+        The redshifts to calculate at.
+    model
+        The model to use for Omega_HI. See `Notes` for details.
+
+    Notes
+    -----
+    We use two models for the evolution of Omega_HI. A fit from Crighton et al. 2015
+    [1]_ (`Crighton2015`) and a fit from the SKA WG whitepaper [2]_ (`SKA`) as well as
+    allowing a constant in redshift value to be set (`uniform`), with a value
+    taken from Switzer et al. 2013 [3]_ assuming :math:`b_{HI}=1`.
```

Comment:
The OmegaHI references disappeared in a previous commit:

```suggestion
    taken from Switzer et al. 2013 [3]_ assuming :math:`b_{HI}=1`.
    
References
----------

.. [1]: https://arxiv.org/abs/1506.02037
.. [2]: https://arxiv.org/abs/1811.02743
.. [3]: https://arxiv.org/abs/1304.3712
```